tensorflow/contrib/gdr/gdr_memory_manager.cc

```diff
@@ -276,8 +275,13 @@ Status GdrMemoryManager::Init() {
     };
     GPUProcessState::singleton()->AddGPUAllocVisitor(bus_id,
                                                      cuda_alloc_visitor);
-    GPUProcessState::singleton()->AddCUDAHostAllocVisitor(bus_id,
-                                                          alloc_visitor);
+
+    for (int numa_idx = 0; numa_idx < port::NUMANumNodes(); ++numa_idx) {
```

Comment:
This should be moved out of the above `if (IsGDRAvailable())`, as the host memory should be registered regardless. Otherwise, it fails for GDR incapable GPUs, such as those GTX cards.

Reply:
Probably should change the name of `IsGDRAvailable` to `P2PKernelModuleEnabled` to avoid confusion.

Any suggestions?

Reply:
GDR is the correct name here. Transporting between GPU's host pinned memory is technically just "GD".

I would like to eventually add the possibility to selectively avoid GPU-direct on slow PCie routes (Like QPI), but I prefer to just focus on hotfixes for now, till the upcoming changes regarding the transport contributions.